scripts/tests/cli-options.mjs

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+import { BrowserStack, DefaultBrowsers } from '../../browsers.config.mjs';
+
+/**
+ * Use standard options for the yargs CLI
+ * @param {Argv} yargs argv
+ * @returns {Argv} yargs argv
+ */
+export const useTestOptions = (argv) => {
```

Comment:
this looks like a builder function. the first param is `yargs` not `argv`

Reply:
If you see the type in yargs it is `Yarg.Argv` interface

Reply:
Renamed to yargs